modules/cert-manager-self-signed-about.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
```

Comment:
@bergerhoffer  Since both self signed and CA issuers do not contain anything specific to openshift and users can directly follow upstream docs, I think we can skip these sections for GA and add them later on if really needed. WDYT?

Reply:
Updating to say we pulled self-signed cert docs for GA. Mentioning in release notes only